src/nn/ops/hard_swish.ts

```diff
@@ -0,0 +1,49 @@
+import * as tf from '@tensorflow/tfjs-core';
+
+import {MLOperand, OutputOperand} from '../operand';
+import {MLOperator, SingleOutputOperation} from '../operation';
+import * as utils from '../utils';
+
+export class HardSwish extends SingleOutputOperation implements MLOperator {
```

Comment:
Could you please check whether it can be implemented by inheriting `UnaryMLOperator`? It is defined in unary.ts. `Relu` and other activation functions are implemented in this way.

Reply:
@huningxin Yes, I checked it could be implemented by inheriting **UnaryMLOperator**, and re-implemented in the latest commit, PTAL, thanks.